sdk/rent-collector/src/lib.rs

```diff
@@ -1,20 +1,23 @@
-#![cfg(feature = "full")]
+//! Calculate and collect rent from accounts.
+
```

Comment:
Can we also have the docsrs bit here?
```suggestion
#![cfg_attr(docsrs, feature(doc_auto_cfg))]
```